ui/app/hooks/useTransactionDisplayData.js

```diff
@@ -145,8 +145,13 @@ export function useTransactionDisplayData (transactionGroup) {
       : initialTransaction.sourceTokenSymbol
     primaryDisplayValue = swapTokenValue
     secondaryDisplayValue = swapTokenFiatAmount
-    prefix = isViewingReceivedTokenFromSwap ? '+' : '-'
-
+    if (isNegative) {
+      prefix = ''
+    } else if (isViewingReceivedTokenFromSwap && !isNegative) {
```

Comment:
Nit: the `!isNegative` here is redundant, as the first branch would always be hit if it was negative

```suggestion
    } else if (isViewingReceivedTokenFromSwap) {
```